modules/nf-core/cellranger/count/tests/main.nf.test.snap

```diff
@@ -2,26 +2,26 @@
     "10x example file": {
         "content": [
             [
-                "versions.yml:md5,3c769adb8bc0f753eb9283b89f6541dd"
+                "versions.yml:md5,0dc3a237aed20a1af22efdadc21222f8"
             ],
             [
-                "filtered_feature_bc_matrix.h5:md5,a875919b9d468efef9f846c95a7f53d6",
```

Comment:
Do we expect this to change? Did you/GCBDS check how different the count matrices are between v8 and v9

Reply:
Hi there, not yet, but I will surely add a comparison.
I was planning in first performing all updates. Then I could focus-compare a few files that draw attention before converting from draft to real PR.
:)